src/Core/Geometry/TriangleMesh.cpp

```diff
@@ -3,6 +3,8 @@
 #include <Core/Geometry/TriangleOperation.hpp> // triangleArea
 #include <Core/Types.hpp>
 
+#include <Engine/Renderer/Mesh/Mesh.hpp>
```

Comment:
This is a bad idea: `Core` must not depend on `Engine`.

Reply:
I agree this is not a good solution. But the way attrib binding is done is also not the better way to do that (attrib name - and then sementic if names are well chosen - is lost and replace by a number). 
So, instead of adding a dependency to the Engine, and as the need is only for the colorize method, why not just write : 
```suggestion
/// TODO : in Radium V2, put here proper attrib binding interface
void TriangleMesh::colorize( const Utils::Color& color ) {
    static const std::string colorAttribName{ "Vec4_attr_0"};
    auto colorAttribHandle = addAttrib<Core::Vector4>( colorAttribName );
    getAttrib( colorAttribHandle ).data() = Vector4Array( vertices().size(), color );
}
```
This is not really clean and depend on the way attribs are managed by the Engine mesh but for now, the color attrib is named `"Vec4_attr_0"` in the Engine mesh.

Reply:
Fixed.